packages/devtools_app/lib/src/debugger/flutter/console.dart

```diff
@@ -61,29 +61,28 @@ class _DebuggerConsoleState extends State<DebuggerConsole> {
     return Console(
       lines: _lines,
       controls: [
-        if (_lines.isNotEmpty) ...[
-          // TODO(ditman) Extract these IconButtons to common_widgets.dart
-          IconButton(
-            icon: const Icon(Icons.content_copy),
-            onPressed: () => copyToClipboard(_lines, context),
-            tooltip: 'Copy to clipboard',
-            key: DebuggerConsole.copyToClipboardButtonKey,
-          ),
-          IconButton(
-            icon: const Icon(Icons.delete),
-            onPressed: widget.controller.clearStdio,
-            tooltip: 'Clear console output',
-            key: DebuggerConsole.clearStdioButtonKey,
-          ),
-        ],
+        // TODO(ditman) Extract these IconButtons to common_widgets.dart
+        IconButton(
+          icon: const Icon(Icons.content_copy, size: actionsIconSize),
+          onPressed:
+              _lines.isEmpty ? null : () => copyToClipboard(_lines, context),
+          tooltip: 'Copy to clipboard',
+          key: DebuggerConsole.copyToClipboardButtonKey,
+        ),
+        IconButton(
+          icon: const Icon(Icons.delete, size: actionsIconSize),
+          onPressed: _lines.isEmpty ? null : widget.controller.clearStdio,
```

Comment:
I did the "don't render disabled buttons" trick because the inkwell animation didn't finish when setting as disabled. I hope that works better now (didn't test it after migrating the whole page to a normal setState flow)

Reply:
Hmm, yeah, that's annoying:

<img width="202" alt="Screen Shot 2020-05-16 at 5 13 56 PM" src="https://user-images.githubusercontent.com/1269969/82132817-ba9e9680-9798-11ea-823a-bb839cd6c437.png">

Any idea how to fix the issue? (also, I would love if the inkwell didn't paint large enough to extend outside the console area...)

Reply:
I didn't find a good solution, let me try and get this failing in dartpad.dev, and shop it around :)

Reply:
All right, two things:

> I would love if the inkwell didn't paint large enough to extend outside the console area

We may be able to wrap the Console in a `Material` widget, that'll prevent the InkWell from overflowing the Console bounds!

Also, the InkWell issue:

https://dartpad.dev/3e21e4a1d81ec137beaeaae3865948d0

I think the problem is the "hover" state of the button is rendered even when disabled (and then it never changes because the mouse listeners are removed). This is probably a framework bug, that is showing in the web, but should show up as well in desktop (anywhere where "hover" makes sense).

I tried working around the issue with something like this:

```
hoverColor: _enabled ? null : Colors.white.withAlpha(0),
```

But when the button gets re-enabled, the "hover" state is shown (it shouldn't!).

I'm 99% sure this is an IconButton issue that is not handling correctly the disabled + hover events.

Reply:
I'll try the `Material` trick -